tasks/install-with-pip.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Install certbot system dependencies
+  package: "name=python3-pip state=present"
+
+- name: Install virtualenv
+  ansible.builtin.pip:
+    name: virtualenv
```

Comment:
This also adds in virtualenv—what if someone wanted to just install certbot systemwide or at the user level?